lib/grandpa/network.go

```diff
@@ -15,10 +16,9 @@ import (
 	"github.com/libp2p/go-libp2p-core/protocol"
 )
 
-var (
-	grandpaID                protocol.ID = "/paritytech/grandpa/1"
-	messageID                            = network.ConsensusMsgType
-	neighbourMessageInterval             = time.Minute * 5
+const (
+	grandpaID1               = "grandpa/1"
+	neighbourMessageInterval = time.Minute * 5
```

Comment:
pedantic nit / out of scope: unit should be at the end


```suggestion
	neighbourMessageInterval = 5 * time.Minute
```